docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
```

Comment:
I would reword this to say this documents a subset of annotations and labels that may be useful to advanced users. I would probably even go so far as to say that any annotations and labels used only internally and _not_ of any particular use to users are omitted.

Reply:
Also, this whole page should move under the users/reference section of the doc tree. Keeping nitty gritty details like these under a reference section is a clear signal to users that it's documentation they can refer to on an as-needed basis and ought not be included in a quick read through of the docs.